casper/src/main/scala/coop/rchain/casper/api/BlockAPI.scala

```diff
@@ -192,7 +193,8 @@ object BlockAPI {
         implicit casper: MultiParentCasper[F]
     ): F[ApiErr[(Seq[DataWithBlockInfo], Int)]] =
       for {
-        mainChain           <- getMainChainFromTip[F](depth)
+        // TODO adjust LFDAN for multiparent or replace with ExploreDeploy
+        mainChain           <- IndexedSeq.empty[BlockMessage].pure
```

Comment:
It's fixed in https://github.com/rchain/rchain/pull/3661/commits/eb783368b6efc0f7a8f32578d9a837120f418449.

Reply:
I did this because there should not be a main chain at all, because this requires Estimator which is obsolete.
@tgrospic thanks, nice workaround, though we still should not have any notion of main chain, is it correct? So am I right that this change is just to make tests happy for now and being able to merge to dev?

Reply:
Yes, no main chain. It's a fix for API to be compatible after HF2. LFDAN should still work the same.